portality/templates/formcontext/maned_journal_review.html

```diff
@@ -141,4 +141,45 @@ <h3 class="heading-aligned-with-fields">Old journal information</h3>
 <script type="text/javascript">
     var notes_deletable = true;
 </script>
+
+<script>
+// When the editor group field is changed, refresh the options for editor
+$("#editor_group").change(function() {
```

Comment:
Is this exactly the same as the one on the application review? If so this is quite easily factored out to a JS file to a named function (in any current JS file that's loaded everywhere, or feel free to make a new one and stick it in _js_includes.html under the correct conditions) and then called on .change.

So this becomes ```$("#editor_group").change(function() {editor_group_changed();}``` .

If there are only small differences between the journal and application .change handlers you can even parametrise them at this point, passing them as function args.